TransformationSystem/Client/FileReport.py

```diff
@@ -1,27 +1,33 @@
-""" FileReport class encapsulates methods to report file status to the transformation DB """
+""" FileReport module defines the FileReport class, to report file status to the transformation DB
+"""
 
-from DIRAC import S_OK, S_ERROR
+import copy
+
+from DIRAC import S_OK
 from DIRAC.Core.Utilities                                     import DEncode
 from DIRAC.TransformationSystem.Client.TransformationClient   import TransformationClient
 from DIRAC.RequestManagementSystem.Client.Operation           import Operation
 
-import copy
-
```

Comment:
if we follow PEP8 import always at the end for example:
from a import b
from a import c

import a